masterlist.yaml

```diff
@@ -5442,27 +5404,13 @@ plugins:
       - name: 'SkyMoMod.esm'
         display: '[Skyrim Monster Mod](http://www.nexusmods.com/skyrim/mods/9694)'
     msg:
-      - type: say
-        content:
-          - lang: en
-            str: 'Obsolete. Update to latest version.'
-          - lang: ru
-            str: 'Устарело. Обновите до последней версии.'
-          - lang: es
-            str: 'Obsoleto. Actualizar a la última versión.'
+      - <<: *obsolete
```

Comment:
I'd get rid of this obsolete message, as the one just a few lines below is more specific.